scripts/zones/Pashhow_Marshlands_[S]/mobs/Nommo.lua

```diff
@@ -1,9 +1,20 @@
 -----------------------------------
 -- Area: Pashhow Marshlands [S]
 --   NM: Nommo
+-- https://www.bg-wiki.com/ffxi/Nommo
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMod(xi.mod.PETRIFYRES, 10000)
+    mob:addMod(xi.mod.MOVE_SPEED_STACKABLE, 25)
```

Comment:
mobs should never have movement modifiers as a form of permanent speed boost. set his speed instead with either his mob family or the lua function to set his speed (the latter only if the family edit negatively impacts mobs that need a different speed)

Reply:
understood, i'll grab the speed from the capture

Reply:
Apologies, Dyinyinga has speed 100 via packet capture, Nommo does not